gui-tk/minsky.tcl

```diff
@@ -273,7 +273,7 @@ if {[tk windowingsystem] == "aqua"} {
     event add <<middleMouse-Motion>> <B3-Motion>
```

Comment:
Interesting - it looks like Macs have already been dealt with on line 271...

Reply:
I noticed that too, which is why it turned out so simple to implement. I had inspiration :)